packages/ui/src/components/Schemas/BarcodeSchema.tsx

```diff
@@ -1,39 +1,20 @@
-  gs1datamatrix: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAANoAAADYCAYAAACN1f3QAAABhGlDQ1BJQ0MgcHJvZmlsZQAAKJF9kT1Iw0AcxV9btSIVQTuICGaoTnZRKY61CkWoEGqFVh1MLv2CJg1Jiouj4Fpw8GOx6uDirKuDqyAIfoC4ujgpukiJ/0sKLWI8OO7Hu3uPu3eAv1FhqtkVB1TNMtLJhJDNrQrBV/RgEAGMISYxU58TxRQ8x9c9fHy9i/Is73N/jn4lbzLAJxDHmW5YxBvEsU1L57xPHGYlSSE+J5406ILEj1yXXX7jXHTYzzPDRiY9TxwmFoodLHcwKxkq8QxxRFE1yvdnXVY4b3FWKzXWuid/YSivrSxzneYokljEEkQIkFFDGRVYiNKqkWIiTfsJD/+I4xfJJZOrDEaOBVShQnL84H/wu1uzMD3lJoUSQPeLbX+MA8FdoFm37e9j226eAIFn4Epr+6sNYPaT9HpbixwBA9vAxXVbk/eAyx1g+EmXDMmRAjT9hQLwfkbflAOGboG+Nbe31j5OH4AMdZW6AQ4OgYkiZa97vLu3s7d/z7T6+wGyunLA/6T1MgAAAAlwSFlzAAAN1wAADdcBQiibeAAAA29JREFUeNrt3VtqLSEURVEN1f8umwYcAiXZYe3oGN+XemZSwvHqHGOs8c588W8qj9X9eDfd69vjudcffA3gzwkNhAZCA4QGQgOhAUIDoQGfno1/W/5recB0bS2u74R73ZqN4osGho4gNEBoIDQQGiA0EBrwaecH63nA/a7CY3X/b/uJZzdD72t2v1dfNDB0BKEBQgOhgdAAoYHQAKFBxFN8vNX8fhOzL1IzSDq/s+6bXFjKAAwdAaGB0EBogNBAaIDQQGhwhjkyMwO6z5b479fW3QnrqFgzBAwdQWiA0EBogNBAaCA0QGjQ3xM677roGVff6wmzam56r9YMAUNHEBogNBAaCA0QGggNEBrkVK8Zkpq10HkNitTMi8SMlHXIMym/Pl80MHQEoQFCA6GB0AChgdAAoUHEc8h9zAPO2X1tkc73mnhn1gwBQ0cQGiA0EBogNBAaCA34pZ2lDBI/Cqd+xK28tuv+235jljIAQ0dAaCA0QGggNBAaIDQQGtxlZymDN7+qpzalGM3Pm7iHzteWer6WMgBDR0BoIDRAaCA0EBogNBAa3GVnZsgJsyUqZ66cMLshdW3rgOvbeg++aGDoCEIDhAZCA6EBQgOhAUKDiJ3dZN4e7xZ2k+lzbYldh6wZAoaOIDRAaCA0QGggNBAaIDTo7wmdNzUzILFmRPdZEKPxeTs/O2uGgKEjCA0QGggNEBoIDYQGCA36S+0m031tkcS9njCDJHVt7Wet+KKBoSMIDRAaCA2EBggNhAYIDSJ2dpOpnC3RfSeWhMSOKH9x3sR7bf/35IsGQgOhAUIDoYHQAKGB0IBPOz9Yp64vIbHJxQnPpPv7j92DLxoYOoLQAKGB0EBogNBAaIDQICK1ycVq/lxm4T1UzyBJnXc2fq+JZ7LVgy8aGDqC0AChgdBAaIDQQGiA0CDiKT7eOuS5lM8MKJSY8TEueq9vn8nWsXzRwNARhAYIDYQGQgOEBkIDhAYR1TNDUrt/pHZ2uclNuw5ZMwQMHQGhgdBAaIDQQGiA0EBocIY5an/x776eReK8K/huu15f6tpm6m/JFw0MHUFogNBAaCA0QGggNEBoELGzZkj5DhuNnbCbTOfznrB2izVDwNARhAYIDYQGCA2EBkIDfql6KQPgszFfNDB0BKEBQgOhgdAAoYHQAKFBxjdAx3fMRvuQ1QAAAABJRU5ErkJggg==',
-};
-
-const SampleBarcode = ({ schema }: { schema: BarcodeSchema }) => (
-  <img
-    style={{
-      width: schema.width * ZOOM,
-      height: schema.height * ZOOM,
-      position: 'absolute',
-      borderRadius: 0,
-      opacity: 0.5,
-    }}
-    src={barcodeExampleImageObj[schema.type]}
-  />
+const SampleBarcode = ({ schema, imageSrc }: { schema: BarcodeSchema, imageSrc: string }) => (
+      <img
```

Comment:
the indents don't look right here, should probably be 2 indents for this line then reduce the others also?